README.md

```diff
@@ -1,5 +1,7 @@
 # answers-headless
-A library for powering UI components for Yext Answers integrations
+A library for powering UI components for Yext Answers integrations.
+
+[Documentation for Exported Types](./docs/answers-headless.md)
```

Comment:
What do you think of renaming this to "Full Documentation"? Not everything that we export is a Type

Reply:
Would "Full Documentation" imply that we're exporting and documenting everything in the repo?

Reply:
I would expect a "Full Documentation" link of repo to provide documentation of the public interface. "Full Documentation" is also the wording we use in Answers Core and it's good to be consistent

Reply:
Ok! I'll change it to that